crates/astria-core/src/protocol/transaction/v1alpha1/action_group/mod.rs

```diff
@@ -149,10 +153,13 @@ enum ErrorKind {
     },
     #[error("attempted to create bundle with non bundleable `ActionGroup` type: {group}")]
     NotBundleable { group: ActionGroup },
+    #[error("attempted to create ActionGroup with empty list of actions")]
```

Comment:
A reader would be confused what `ActionGroup` the error is talking about because fundamentally this is about the actions contained in a transaction not being empty:

```suggestion
#[error("actions cannot be empty")]
```